packages/frontend-defaults/report.api.md

```diff
@@ -32,6 +32,8 @@ export interface CreateAppOptions {
     config: ConfigApi;
   }>;
   // (undocumented)
+  featureLoaderRecursionDepth?: number;
```

Comment:
Meh, I don't think this is worth an option, let's just hardcode it at somewhere around 5-10 and leave it at that?

Reply:
Quite useful for tests though :-)

Reply:
Which is why I made it optional

Reply:
Fixed.